src/Raven.Server/Json/BlittableJsonTextWriterExtensions.cs

```diff
@@ -1412,17 +1429,17 @@ public static void WriteObjects(this BlittableJsonTextWriter writer, JsonOperati
             writer.WriteEndArray();
         }
 
-        public static async Task<int> WriteObjectsAsync(this AsyncBlittableJsonTextWriter writer, JsonOperationContext context, IEnumerable<BlittableJsonReaderObject> objects)
+        public static async Task<(int numberOfResults, long totalDocumentsSizeInBytes)> WriteObjectsAsync(this AsyncBlittableJsonTextWriter writer, JsonOperationContext context, IEnumerable<BlittableJsonReaderObject> objects)
```

Comment:
Let's upper case first letters - `NumberOfResults` and `TotalDocumentsSizeInBytes`

Then if you do:

```
var result = await WriteObjectsAsync();
```

then you have `result.NumberOfResults` instead of `result.numberOfResults`.

Reply:
done - no need to update the calls